mindsdb/integrations/handlers/together-ai-handler/together_ai_handler.py

```diff
@@ -0,0 +1,86 @@
+import os
+from typing import Optional, Dict
+
+import together
+from concurrent.futures import ThreadPoolExecutor
+import pandas as pd
+
+from mindsdb.utilities.config import Config
+from mindsdb.integrations.libs.base import BaseMLEngine
+from mindsdb.integrations.libs.llm_utils import get_completed_prompts
+
+
+class TogetherAIHandler(BaseMLEngine):
+    """
+    Integration with the Together AI Inference API.
+    """
+
+    name = 'together_ai'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.default_model = 'togethercomputer/RedPajama-INCITE-7B-Chat'
+        self.default_max_tokens = 128
+
+    def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[Dict] = None) -> None:
+        if 'using' not in args:
+            raise ValueError("Together AI engine requires a USING clause! Refer to its documentation for more details.")
+        # set api key
+        api_key = self._get_together_ai_api_key(args['using'])
```

Comment:
You probably want to do `args['using']['api_key']` instead, as `args['using']` is an entire dictionary of parameters.